### PR TITLE
Styling flash messages 35

### PR DESCRIPTION
**Pull request by meyerhoferc**
all tests pass

* adds green styling to flash messages
* adds materialize styling to buttons
* adds styling to cart in table headers & text in cells

### CODE REVIEW

File: app/views/carts/show.html.erb [hunk 1]
```diff
@@ -4,33 +4,33 @@
...
+        <th data-field="price"><h5>Price</h5></th>
+        <th data-field="price"><h5>Subtotal</h5></th>
+        <th data-field="remove"><h5>Remove</h5></th>
     </tr>
   </thead>
 
   <tbody>
     <% @cart.contents.each do |item, quantity| %>
     <tr>
       <td><img src="<%=Item.find(item).image_tag%>"/></td>
```
Comment (cmacaulay) [inline app/views/carts/show.html.erb:20]: Do we want to take this AR logic out of this view somehow? @meyerhoferc @CjMoore @valerietrudell

File: app/views/carts/show.html.erb [hunk 2]
```diff
@@ -4,33 +4,33 @@
...
         <% end %>
       </td>
-      <td>$<%= Item.find(item).price %></td>
-      <td>$<%= @cart.subtotal(item) %></td>
-      <td><%= link_to "Remove Item", cart_path({item_id: item}), method: :delete%><td>
+      <td><h5>$<%= Item.find(item).price %></h5></td>
+      <td><h5>$<%= @cart.subtotal(item) %></h5></td>
+      <td><h5><%= link_to "Remove Item", cart_path({item_id: item}), method: :delete%></h5><td>
     </tr>
     <% end %>
```
Comment (cmacaulay) [inline app/views/carts/show.html.erb:35]: 👍 I like your button

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -14,12 +14,12 @@
 
   <body>
     <%= render partial: 'shared/nav' %>
-    <div class="container">
+    <div class="container flash">
```
Comment (cmacaulay) [inline app/views/layouts/application.html.erb:17]: nice lookin' class

**Review by cmacaulay — APPROVED** [review]
Looks good to me! Wondering if we need to figure out a way to remove the active record logic from the cart show view?

File: app/views/carts/show.html.erb (hunk 1 above)
Comment (CjMoore) [reply app/views/carts/show.html.erb:20]: I would say yes. I'll make a card for refactoring it.

**Comment by CjMoore** [comment]
I added a card to refactor the AR in the cart show page. Otherwise this looks great.

**Pull request merged**